products/idn/tools/sdk/go.md

```diff
@@ -0,0 +1,387 @@
+---
+id: go-sdk
+title: Go SDK
+pagination_label: Go SDK
+sidebar_label: Golang
+sidebar_position: 2
+sidebar_class_name: gosdk
+keywords: ['go', 'golang', 'sdk']
+description: Learn how to use the Golang SDK in this guide. 
+slug: /tools/sdk/go
+tags: ['SDK', 'Software Development Kit']
+---
+
+## Start using the Go SDK
```

Comment:
Should each of these individual SDK docs have links to the SDKs somewhere right at the top? I had to look around for a while before I found where I could just get the SDK. (This also goes for Powershell and Typescript).